account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/client/SignerClient.java

```diff
@@ -0,0 +1,218 @@
+     * @param txBytes
+     *            keccack256 hashed transaction byte
+     *
+     * @return
+     *
+     * @throws Exception
+     */
+    public String signEthTx(Stamper stamper, User user, Bytes txBytes) throws Exception {
+        return this.signRawMessage(stamper, user, txBytes, SigningMode.ETHEREUM, "HASH_FUNCTION_NO_OP", user.address);
+    }
+
+    public String targetPublicKeyHex(Stamper stamper) throws GeneralSecurityException, InvalidProtocolBufferException {
+        return stamper.publicKey();
+    }
+
+    public String targetPublicKeyJwtNonce(Stamper stamper)
+            throws GeneralSecurityException, InvalidProtocolBufferException {
+        MessageDigest digest = MessageDigest.getInstance("SHA-256");
+        byte[] hash = digest.digest(stamper.publicKey().getBytes(StandardCharsets.UTF_8));
+        String sha256hex = new String(Hex.encode(hash));
+
+        return stamper.publicKey();
+    }
+
+    private String getUserOrgId(String email) throws Exception {
+        GetUser.Request getUserRequest = new GetUser.Request(email);
+        GetUser.Response getUserResponse = this.request(PathName.LOOKUP.getName(), getUserRequest,
+                GetUser.Response.class);
+        return getUserResponse.orgId();
+    }
+
+    /**
+     * Auth user with Stamp.
+     *
+     * @param orgId
+     *
+     * @return
+     *
+     * @throws Exception
+     */
+    private User authUser(Stamper stamper, String orgId) throws Exception {
+
+        WhoAmIRequest whoAmIRequest = new WhoAmIRequest(orgId);
+        ObjectWriter writer = this.mapper.writerWithDefaultPrettyPrinter();
+        String json_body = writer.writeValueAsString(whoAmIRequest);
+        Stamp stampedBody = stamper.stamp(json_body);
+        StampedRequest stampedRequestrequest = StampedRequest.builder().url("https://api.whoami.com/v1/users/").body(json_body)
+                .stamp(stampedBody).build();
+        TurnKeyWhoAmIRequest request = new TurnKeyWhoAmIRequest(stampedRequestrequest);
+        AuthUser.Response response = this.request(PathName.WHOAMI.getName(), request, AuthUser.Response.class);
+        return User.builder().address(response.address()).orgId(response.orgId()).userId(response.userId())
+                .email(response.email()).solanaAddress(response.solanaAddress()).build();
+
+    }
+
+    private <Request, Response> Response request(String path, Request request, Class<Response> clazz) throws Exception {
+
+        URI uri = URI.create(this.httpConfig.getUrl()).resolve(path);
+        HttpRequest http_request = HttpRequest.newBuilder().uri(uri).header("accept", "application/json")
+                .header("content-type", "application/json")
+                .header("Authorization", "Bearer " + this.httpConfig.getApiKey())
+                .method("POST", HttpRequest.BodyPublishers.ofString(this.mapper.writeValueAsString(request))).build();
+
+        JacksonBodyHandlers jsonBodyHandler = new JacksonBodyHandlers(this.mapper);
```

Comment:
an we remove JacksonBodyHandlers? It doesn't seem to be used here / feels unnecessary?